src/wallet/scriptpubkeyman.cpp

```diff
@@ -1912,10 +1912,10 @@ bool DescriptorScriptPubKeyMan::SetupDescriptorGeneration(const CExtKey& master_
     if (Params().IsTestChain()) {
         desc_prefix += "/1'";
     } else {
-        desc_prefix += "/0'";
+        desc_prefix += "/3'";
     }
 
-    std::string internal_path = m_internal ? "/1" : "/0";
+    std::string internal_path = m_internal ? "/1" : "/3";
```

Comment:
**Important:** This element in the path should not change, it's a different element (see line 1919 for how it's constructed)

See [BIP-0044's "path-levels"](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#path-levels) - this is the "change" element.